platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/platform/StartupGenerator.xtend

```diff
@@ -36,7 +36,7 @@ class StartupGenerator implements IPlatformStartupGenerator {
 		
 		codeFragmentProvider.create('''
 		/* Mapping Default Error Handling function */
-		Retcode_T returnValue = Retcode_initialize(DefaultErrorHandlingFunc);
+		Retcode_T returnValue = Retcode_Initialize(DefaultErrorHandlingFunc);
```

Comment:
Are you using an old SDK? I'm pretty sure the current code is correct.

Reply:
we are using the current xdk110 master, deprecated warnings were coming so fixed it

Reply:
Still, this should be a separate thing. If the current implementation does not represent the shipped SDK we need to address that in a separate issue and merge request.

Reply:
i will check and revert.